uast/nodes/node.go

```diff
@@ -948,3 +948,28 @@ func UniqueKey(n Node) Comparable {
 		return unkPtr(ptr)
 	}
 }
+
+// ChildrenCount returns a number of immediate children nodes. For arrays it will be the
+// length of an array. For objects, all child Object and Array nodes in fields will be counted.
```

Comment:
I was confused (until I read the code) about whether this means the Array is counted as 1 (for itself) or for its children. May I suggest

```
// ChildrenCount reports the number of immediate children of n. If n is an Array this is
// the length of the array. If n is an object, each object or array in a field of n counts as 
// one child.
```

or words to that effect?

Reply:
Changed the comment, PTAL.